src/main/resources/com/dynatrace/jenkins/dashboard/steps/TAReportingRecorderPipelineStep/help.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  This plugin fetches analysis data from the <a href="http://www.dynatrace.com">Dynatrace Server</a> (Test Center Edition) REST
```

Comment:
1) AppMon server
2) Change url to AppMon doc: https://www.dynatrace.com/support/doc/appmon/